### ISSUE TITLE
is the code open source?

### ISSUE THREAD

**Issue by caleb15** [dlopes7/dynatrace-csv-wrapper#1]
I was about to look at it and then I realized that the repo doesn't have a license.

**Comment by dlopes7** [comment]
Yeah but it is a huge mess to be honest, I would stick with the standard csv content type format of the metrics v2 API

**Comment by caleb15** [comment]
ok, thanks
